charts/civil-ga-ccd/values.template.yaml

```diff
@@ -258,6 +258,7 @@ ccd:
       keyVaults: []
   ccd-definition-store-api:
     java:
+      releaseNameOverride: ${SERVICE_NAME}-ccd-definition-store-api
```

Comment:
these changes in values.template.yaml are not required since values elastic search has got those